CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_netsuite_source v0.5.1
+
+## Features 🎉 (affects Netsuite2 users only)
+- Introduces the `netsuite2__multibook_accounting` and `netsuite2__using_vendor_categories` variables to disable their related source tables and downstream models ([#21](https://github.com/fivetran/dbt_netsuite_source/pull/21)).
+  - `netsuite2__multibook_accounting` is `True` by default. Set it to `False` if you do not use the [Multi-Book Accounting](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/book_3831565332.html) feature in Netsuite and/or do not have the `accountingbook` and `accountingbooksubsidiaries` source tables.
```

Comment:
This is more of a stylistic question, but I'm curious why you opted for `netsuite2__multibook_accounting` instead of `netsuite2__using_multibook_accounting`?

Reply:
i did this since there's no `multibook_accounting` table per se, but wasn't sure if that was more or less confusing

Reply:
changing to `netsuite2__multibook_accounting_enabled`